app/src/main/java/com/imashnake/animite/features/searchbar/SearchFrontDrop.kt

```diff
@@ -184,29 +186,26 @@ private fun SearchItem(
             )
             if (item.seasonYear != null) {
                 Text(
-                    text = item.seasonYear,
+                    text = item.seasonYear.toString(),
                     color = MaterialTheme.colorScheme.onBackground,
                     style = MaterialTheme.typography.labelSmall
                 )
             }
 
             Spacer(Modifier.size(dimensionResource(R.dimen.medium_padding)))
 
-            if (item.studios != null) {
-                Text(
-                    text = item.studios,
-                    color = MaterialTheme.colorScheme.onBackground,
-                    style = MaterialTheme.typography.bodyMedium,
-                    maxLines = 1,
-                    overflow = TextOverflow.Ellipsis
-                )
-            }
             Text(
-                text = item.footer(
-                    if (item.episodes != null) {
-                        pluralStringResource(R.plurals.episodes, item.episodes, item.episodes)
-                    } else null
-                ),
+                text = item.studios.joinToString(", "),
```

Comment:
`joinToString`'s default separator is already `", "` - https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/join-to-string.html

Reply:
Nice!